### PR TITLE
Update readmes in the repo.

### PR DESCRIPTION
**Pull request by Ragansu**
Added READMEs to ingestion, scoring , sample submission and test model
Also corrected a typo in README of conda 

RC

### CODE REVIEW

File: ingestion_program/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+--systematics-diboson-scale \ 
+--systematics-bkg-scale \
+--num-pseudo-experiments 100 \ 
+--num-of-sets 1 
+```
+
+If you have GPU you can use the `--parallel` flag to parallelize the pseudo experiments.
+
+For more information on flags run `python3 run_ingestion.py -h`
+For more information on the workings of the ingestion program check out our [whitepaper](https://example.com/whitepaper)
```
Comment (ihsaan-ullah) [inline ingestion_program/README.md:26]: you can put a pdf link where we can put the white paper
https://fair-universe.lbl.gov/whitepaper.pdf

File: sample_code_submission/README.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# Sample code submission
+This folder contains the sample code submission
+## Description
+The main part of the sample code submission is in the `model.py` which contains the `Model` class. 
+* The model class uses the BDT classifier using [xgboost](https://xgboost.ai/) defined in [boosted_decision_tree](/sample_code_submission/boosted_decision_tree.py). 
+* The statistical analysis part is written in [statistical_analysis](/sample_code_submission/statistical_analysis.py) 
+
+Workings of the Statistical Analysis part can be found in our [whitepaper](https://example.com/whitepaper)
```
Comment (ihsaan-ullah) [inline sample_code_submission/README.md:8]: you can put a pdf link where we can put the white paper
https://fair-universe.lbl.gov/whitepaper.pdf

File: scoring_program/README.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+## Output
+The program will output the calculated scores, including 
+1. Mean Average Estimator (MAE)
+2. Root Mean Square Estimator (RMSE)
+3. Quantile Score (QS)
+
+The detailed coverage plots will be available in the [detailed_results.html](/scoring_output/detailed_results.html)
+
+For more information on the workings of the scoring program check out our [whitepaper](https://example.com/whitepaper)
```
Comment (ihsaan-ullah) [inline scoring_program/README.md:24]: you can put a pdf link where we can put the white paper
https://fair-universe.lbl.gov/whitepaper.pdf

**Comment by Ragansu** [comment]
Okay I will update that

**Review by ihsaan-ullah — APPROVED** [review]

**Pull request merged**